features/support/object_model_pages/your_information.rb

```diff
@@ -27,4 +27,8 @@ def self.welcome_sign_in_message
   def self.continue_btn
     '.interaction-click-control-continue'
   end
+
+  def self.interactive_verifications_page
+    "Upload document(s) proving your identity"
```

Comment:
please add a css class/attribute here instead of a hardcoded string of english text

Reply:
for this we don't have any specific css class/attribute except ul, li.

Reply:
@harshared please add a class or add an attribute like `data-cucumber="upload-document"` for use in this selector

Reply:
@markgoho just curious what's wrong in using this hardcoded english text in this POM, its global and reusable

Reply:
@harshared because it's brittle:
1. if the text ever changes the spec fails
2. if the test is ever run in a different translation, the spec fails

choosing something that isn't likely to ever change (a css selector, or even better a specific attribute on an element) is the way to go here and in all places

Reply:
@markgoho just curious why exactly are you looking to add a class or add an attribute to test here.

Reply:
@harshared I just answered that question in the previous comment

Reply:
@markgoho but for suppose if the developer by mistake enters a wrong text in the html.erb file then we don't have any source to test that if we use that css selector, or even better a specific attribute on an element)
And I think that's one of the reason why we are using this test cases right
and if the text changes then the developer has to change its respective step defenition for it to pass

Reply:
@harshared what you're identifying as a potential pitfall is, in fact, the flexibility that writing tests without hardcoded english text allows

please:
1. add a css class or `data-` attribute to the element you're searching for and
2. adjust the POM reference to that newly-added class

Reply:
addressed